.github/workflows/cd.yml

```diff
@@ -127,6 +127,11 @@ jobs:
       - name: generate templates v3
         run: |
           .github/scripts/template-zip-autogen-v3.sh ${{ runner.temp }}/teamsfx_templates_v3
```

Comment:
the input ${{ runner.temp }}/teamsfx_templates_v3 is duplicate with env.TEMPLATE_PATH, can merge together?

Reply:
Good suggestion. Updated.